src/icons/fileIcons.ts

```diff
@@ -430,7 +430,7 @@ export const fileIcons: FileIcons = {
         { name: 'vue', fileExtensions: ['vue'] },
         { name: 'vue-config', fileNames: ['vue.config.js', 'vue.config.ts'] },
         { name: 'vuex-store', fileExtensions: ['store.js', 'store.ts'], fileNames: ['store.js', 'store.ts'], enabledFor: [IconPack.Vuex] },
-        { name: 'nuxt', fileNames: ['nuxt.config.js', 'nuxt.config.ts'] },
+        { name: 'nuxt', fileNames: ['nuxt.config.js', 'nuxt.config.ts', '.nuxtignore'] },
```

Comment:
Something went wrong here? You included the changes of PR #995? Could you please check that? I'm excited about the Jest folder icon, we could really use that.

Reply:
Oh, I'm my bad. It was just a request for the icon. I am not sure why I am included in these changes, as far as I know, I didn't change any code. I hope I didn't cause you any problems.

Reply:
No worries, it didn't cause any problems. If it's only an icon request you are always welcome to create an issue for this. Pull Requests are intended to merge some new code into the existing code base. Would be nice if you could create a new issue for this. I will close this PR now.